python/dgl/graphbolt/utils/sample_utils.py

```diff
@@ -267,3 +268,110 @@ def unique_and_compact_node_pairs(
         unique_nodes = list(unique_nodes.values())[0]
 
     return unique_nodes, compacted_node_pairs
+
+
+def compact_node_pairs(
```

Comment:
Maybe rename it to compact_csc_format?

Reply:
And add a comment at L136 in python/dgl/graphbolt/impl/neighbor_sampler.py to clarify why we assign compacted_csc_format to node_pairs.